CONTRIBUTING.md

```diff
@@ -81,6 +82,55 @@ $ npx prettier * --write
 
 This should be done before opening a PR to keep checked-in code consistent.
 
+### Onboarding
+
+To onboard your release tests (Dashboards plugins) onto this repo test function, you can follow the next steps.
+
+0. Create a name to identify your plugin, e.g `plugin-name`
+
+The dir name shall be descriptive to identify your plugin. You can use the same name defined in build repo https://github.com/opensearch-project/opensearch-build/tree/main/scripts/components
```

Comment:
any question to enforce same folder name as same plugin name?

Reply:
No strong reason. Just make it a bit easy to correlate to each other.

Reply:
Sorry, I mean why not enforce to use same name